MapboxCoreNavigation/NavigationServiceDelegate.swift

```diff
@@ -139,6 +139,16 @@ public protocol NavigationServiceDelegate: class, UnimplementedLogging {
      */
     func navigationService(_ service: NavigationService, shouldPreventReroutesWhenArrivingAt waypoint: Waypoint) -> Bool
     
+    /**
+     Called when the location manager's accuracy authorization changed.
+     
+     You can implement this method to allow the navigation service to check if the user changed accuracy authorization, especially if reducedAccuracy is enabled. This method is only relevant for iOS 14 and above.
+     
+     - parameter service: The navigation service that will alert that user that reducedAccuracy is enabled.
+     - parameter manager: The location manager.
+     */
+    func navigationServiceDidChangeAuthorization(_ service: NavigationService, manager: CLLocationManager)
```

Comment:
Nit: Since there’s a second argument, it’s more natural to write `navigationService(_:didChangeAuthorizationFor:)`.

```swift
func navigationService(_ service: NavigationService, didChangeAuthorizationFor locationManager: CLLocationManager) {
```